### PR TITLE
Syncjog GET : Add dates

### PR DESCRIPTION
**Pull request by snimavat**

### CODE REVIEW

**Comment by snimavat** [comment]
@basejump

SyncJobCrudApi overrides `entityToMap` and calls `JobUtils.convertToMap(job)`
so default implementation which creates meta map from includes doesnt work

**Comment by snimavat** [comment]
Done

File: gorm-rest/src/main/groovy/yakworks/rest/gorm/render/SyncJobRenderer.groovy [hunk 1]
```diff
@@ -21,6 +21,7 @@ import grails.rest.render.RenderContext
  */
 @Slf4j
 @CompileStatic
+//XXX is this used now, since we have SyncjobCrudApi
```
Comment (basejump) [inline gorm-rest/src/main/groovy/yakworks/rest/gorm/render/SyncJobRenderer.groovy:24]: 👀

File: gorm-tools/src/main/groovy/gorm/tools/job/JobUtils.groovy [hunk 2]
```diff
@@ -22,11 +22,15 @@ class JobUtils {
...
-        JsonOutput.JsonUnescaped rawDataJson = JsonOutput.unescaped(jobData)
+        return JsonOutput.unescaped(jobData)
+    }
+
+    //static helper as its used both here and also in the SyncJobRenderer
+    static Map convertToMap(SyncJobEntity job){
+        JsonOutput.JsonUnescaped rawDataJson = getRowJobData(job)
 
         Map map = [
             id: job['id'],
```
Comment (basejump) [inline gorm-tools/src/main/groovy/gorm/tools/job/JobUtils.groovy:36]: 👀 look at original code, do we have every field? I think we are missing tests for "problem"

**Review by basejump — CHANGES_REQUESTED** [review]

**Review by jdabal — APPROVED** [review]

**Pull request merged**